OutputHelper.cs

```diff
@@ -0,0 +1,44 @@
+﻿using NLog;
```

Comment:
Nice refactor! This file looks like it has a specific purpose and is a good collection of related functions 🙂